### ISSUE TITLE
NTR: 6-mercaptopurine sensitivity (Triggers or responses: drugs or treatments)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9494]
New term request 6-mercaptopurine sensitivity (segal_120425154029): 6-MP; 6MP; thiopurines

**Comment by pnrobinson** [comment]
@MickeySegal
Is this referring to https://pubmed.ncbi.nlm.nih.gov/36120820/ ?
Please provide more context.

**Comment by MickeySegal** [comment]
Correct.  The drug 6-mercaptopurine, typically given for immune disorders or blood cancers, is not broken down with the normal speed and has toxic effects, typically noticed by effects on blood cells.  See also https://omim.org/entry/610460
